Resources/Prototypes/DeltaV/Entities/Objects/Specific/Security/security.yml

```diff
@@ -0,0 +1,26 @@
+- type: entity
+  parent: Stunbaton
+  id: StunBatonSelfCharging
+  suffix: Recharging, Do Not Map
+  components:
+  - type: BatterySelfRecharger
+    autoRecharge: true
+    autoRechargeRate: 50
+
+- type: entity
+  parent: Flash
+  id: FlashRecharging
+  suffix: Recharging, Do Not Map
+  components:
+  - type: AutoRecharge
+    rechargeDuration: 120
+
+#- type: entity - This is dumb, can't figure it out, will come back when more patient
```

Comment:
im pretty sure you cant combine a stack-less item into a stack item
so youd have to add stack to handcuffs with max stack of 1

Reply:
i've just seen this on https://github.com/space-wizards/space-station-14/pull/21349 , i will revisit this soon but this is also mega useful thank you so much i should probably also reading how lingering & stack components work when i come back to this